tools/esdocgen/Makefile

```diff
@@ -7,6 +7,6 @@ all: esdocgen.cmo esdocgen.cmxs
 	ocamlopt -I +ocamldoc -shared -o $@ $<
 
 clean:
-	rm -f *.{cm{o,i,x,xs},o} || true
+	rm -f *.cmo *.cmi *.cmx *.cmxs *.o
```

Comment:
The `||true` is not needed?

Reply:
I think it was a MacOSX not supporting the `-f` flag hack, similar to the `sh -c "rm -f ..."` hacks elsewhere. Another alternative is to prefix the command line with an `-`, which tells make to ignore failures.

Reply:
(The `-f` flag causes `rm` to not return an error if none of the files to be deleted exist when attempted.)